Libplanet.Tests/Assets/CurrencyTest.cs

```diff
@@ -87,25 +87,25 @@ public void Constructor()
         [Fact]
         public void Hash()
         {
-#pragma warning disable CS0612
-            Currency currency = Currency.LegacyUntracked("GOLD", 2, AddressA);
+#pragma warning disable CS0618
+            Currency currency = Currency.Legacy("GOLD", 2, AddressA);
             HashDigest<SHA1> expected =
                 HashDigest<SHA1>.FromString("81446cd346c1be9e686835742bfd3772194dea21");
             AssertBytesEqual(expected, currency.Hash);
 
             currency =
-                Currency.LegacyUntracked("NCG", 8, ImmutableHashSet.Create(AddressA, AddressB));
+                Currency.Legacy("NCG", 8, ImmutableHashSet.Create(AddressA, AddressB));
             expected = HashDigest<SHA1>.FromString("42ce3a098fe14084e89d3d4449f56126693aeed1");
             AssertBytesEqual(expected, currency.Hash);
 
-            currency = Currency.LegacyUntracked("FOO", 0, ImmutableHashSet<Address>.Empty);
+            currency = Currency.Legacy("FOO", 0, ImmutableHashSet<Address>.Empty);
             expected = HashDigest<SHA1>.FromString("801990ea2885bd51eebca0e826cc0e27f0917a9b");
             AssertBytesEqual(expected, currency.Hash);
 
-            currency = Currency.LegacyUntracked("BAR", 1, null);
+            currency = Currency.Legacy("BAR", 1, null);
             expected = HashDigest<SHA1>.FromString("da42781871890f1e1b7d6f49c7f2733d3ba7b8bd");
             AssertBytesEqual(expected, currency.Hash);
-#pragma warning restore CS0612
+#pragma warning restore CS0618
```

Comment:
Same here (comment to justify, and narrowing its scope).

IMHO, it's better to suppress CS0618 for each single line instead of a single block of lines.